smart-contracts/assembly/contracts/pool.ts

```diff
@@ -652,7 +618,12 @@ function _getFeeRate(): f64 {
  * @returns The current fee share for the protocol.
  */
 function _getFeeShareProtocol(): f64 {
```

Comment:
Remove this and get Protocol fees in constructor. Fix the protocol fee in Registry too.